locomotion-sdk/src/pages/Account/index.js

```diff
@@ -144,18 +139,15 @@ export default ({
           title={i18n.t('onboarding.pageTitle')}
           onIconPress={() => navigation.toggleDrawer()}
           iconSide={menuSide}
+        />
+        <AccountHeader />
+        <LogoutContainer
+          onPress={() => {
+            navigation.navigate(ROUTES.LOGOUT);
+          }}
         >
-          <>
-            <AccountHeader />
-            <LogoutContainer
-              onPress={() => {
-                navigation.navigate(ROUTES.LOGOUT);
-              }}
-            >
-              <ErrorText>{i18n.t('menu.logout')}</ErrorText>
-            </LogoutContainer>
-          </>
-        </PageHeader>
```

Comment:
no header?

Reply:
moved it to the other component instead of a child